### PR TITLE
1 jest typescript node

### LINKED ISSUE #1: Install NodeJs, Typescript, Jest

### PR DESCRIPTION
**Pull request by DanielMJSilva**
@rmc3408 can you review?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+
+const app = express()
+const port = 3000
```
Comment (rmc3408) [inline index.js]: missing Typescript codes.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const express = require('express')
```
Comment (rmc3408) [inline index.js]: use import

**Review by rmc3408 — APPROVED** [review]
LGTM, i might only to implement Types. But feel free to merge and i do on next task. I will do it tonight.

**Comment by DanielMJSilva** [comment]
I did this on #1 
can I close this task? @rmc3408

**Comment by rmc3408** [comment]
not close issue ... I mean approved then it merge to develop branch.

**Review by rmc3408 — APPROVED** [review]
LGTM

**Review by DanielMJSilva — COMMENTED** [review]
@rmc3408 can you review?

**Comment by rmc3408** [comment]
done

**Review by rmc3408 — APPROVED** [review]
LGTM

**Pull request merged**